### DISCUSSION TITLE
enable postgresql / redis / tika / gotenberg server selection during paperless-ngx install + add postgresl access

### DISCUSSION THREAD

**Discussion by jsapede** [tteck/Proxmox#3562] [Ideas]
changing settings for redis, tika and gotenberg is quite easy after install however database export is not so quite easy.
would be useful to have the ability to choose postgresql server at installation

moreover postgresql seems not accessible from commandline ot export database only to another external server, seems permissions are very restrictive on connexion

**Comment by tteck** [comment]
> would be useful to have the ability to choose postgresql server at installation

PostgreSQL is the default database. Built in for single backup (can be changed in `/opt/paperless/paperless.conf`).

https://github.com/tteck/Proxmox/blob/de1a4b0914c63849912cb5187117191b8eeba1b0/install/paperless-ngx-install.sh#L17-L19
https://github.com/tteck/Proxmox/blob/de1a4b0914c63849912cb5187117191b8eeba1b0/install/paperless-ngx-install.sh#L100-L122

Adminer is an option to administer the database 

https://github.com/tteck/Proxmox/blob/de1a4b0914c63849912cb5187117191b8eeba1b0/install/paperless-ngx-install.sh#L124-L139

**Reply by jsapede** [reply]
Sorry maybe i have not been clear with thé suggestion

The point would be to have the ability to choses at script exécution the IP for the psql serveur and then avoid to reinstall a specific instance it if you already have a running psql server

As well as redis
